eng/Versions.props

```diff
@@ -63,6 +63,7 @@
     <SystemTextEncodingsWebVersion>9.0.0</SystemTextEncodingsWebVersion>
     <SystemNumericsTensorsVersion>9.0.0</SystemNumericsTensorsVersion>
     <SystemTextJsonVersion>9.0.0</SystemTextJsonVersion>
+    <SystemThreadingChannelsVersion>9.0.0</SystemThreadingChannelsVersion>
```

Comment:
@stephentoub, I just stumbled on this change. This won't work as you expect. For these packages to be updated via the automated dependency flow there must be an associated entry in Version.Details.xml.
The maestro bot is querying Version.Details.xml to determine what dependencies must be updated.